database/ongoing/unsubscribe-users.js

```diff
@@ -0,0 +1,38 @@
+const getItems = require('../utils/get-items');
+const asyncForEach = require('../utils/async-for-each');
+const { logger, generateProgressBar } = require('../utils/logging');
+
+module.exports = {
+  async up(db, client) {
+    const emails = getItems('bouncing-emails.csv');
+    const progress = generateProgressBar(emails.length);
+    await asyncForEach(emails, async (email, index) => {
+      const log = (m, p, t = 'info') => logger(t, index, m, p);
+      log('Query out: ', email);
+      progress.tick();
+      await db
+        .collection('user')
+        .updateOne(
```

Comment:
Can there be more than one user with the same email?

Reply:
Yes, there can and will be. But we always operate on the first found record (everywhere, using the default sort, including the API). So far, our strategy has been to consolidate duplicates as reports come in (by merging only the progress onto the first found record)

Reply:
I see, presumably if there are multiple bouncing emails they'll get caught by subsequent runs.